src/components/media-player/dialog-media-player-group.ts

```diff
@@ -0,0 +1,155 @@
+
+const includeDomains = ["media_player"];
+
+@customElement("dialog-media-player-group")
+class DialogMediaPlayerGroup extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _entityId?: string;
+
+  @state() private _groupMembers!: string[];
+
+  @state() private _submitting?: boolean;
+
+  public showDialog(params: MediaPlayerGroupDialogParams): void {
+    this._entityId = params.entityId;
+
+    const stateObj = this.hass.states[params.entityId] as
+      | (HassEntity & { attributes: { group_members?: string[] } })
+      | undefined;
+    this._groupMembers =
+      stateObj?.attributes.group_members?.filter(
+        (entityId) => entityId !== params.entityId
+      ) || [];
+  }
+
+  public closeDialog() {
+    this._entityId = undefined;
+    this._groupMembers = [];
+    this._submitting = false;
+    this.classList.remove("opened");
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  protected render() {
+    if (!this._entityId) {
+      return nothing;
+    }
+
+    const entityId = this._entityId;
+    const stateObj = this.hass.states[entityId] as HassEntity | undefined;
+    const name = (stateObj && computeStateName(stateObj)) || entityId;
+
+    return html`
+      <ha-dialog
+        open
+        scrimClickAction
+        escapeKeyAction
+        flexContent
+        .heading=${name}
+        @closed=${this.closeDialog}
+        @opened=${this._dialogOpened}
+      >
+        <ha-dialog-header show-border slot="heading">
+          <span slot="title"> ${name} </span>
+          <ha-icon-button
+            .label=${this.hass.localize("ui.common.close")}
+            .path=${mdiClose}
+            dialogAction="close"
+            slot="actionItems"
+          ></ha-icon-button>
+        </ha-dialog-header>
+        <ha-entity-picker
+          .hass=${this.hass}
+          .value=${this._entityId}
+          disabled
+          required
+          hide-clear-icon
+        >
+        </ha-entity-picker>
+        <ha-entities-picker
+          .hass=${this.hass}
+          .value=${this._groupMembers}
+          .autofocus=${true}
+          .includeDomains=${includeDomains}
+          .entityFilter=${this._filterEntities}
+          .pickedEntityLabel=${this.hass.localize(
+            "ui.panel.config.person.detail.device_tracker_picked"
+          )}
+          .pickEntityLabel=${this.hass.localize(
+            "ui.panel.config.person.detail.device_tracker_pick"
+          )}
+          @value-changed=${this._groupMembersChanged}
+        >
+        </ha-entities-picker>
+        <mwc-button slot="secondaryAction" @click=${this.closeDialog}>
+          ${this.hass.localize("ui.common.close")}
+        </mwc-button>
+        <mwc-button
+          .disabled=${this._submitting}
+          slot="primaryAction"
+          @click=${this._submit}
+        >
+          ${this.hass.localize("ui.common.save")}
+        </mwc-button>
```

Comment:
Use the ha-button, so we import the css definitions of the home assistant buttons. 

```suggestion
        <ha-button
          .disabled=${this._submitting}
          slot="primaryAction"
          @click=${this._submit}
        >
          ${this.hass.localize("ui.common.save")}
        </ha-button>
```

Add an import for ha-button like: 
```
import "../ha-button";
```